Начинаем программировать

faviru opened the pull request:
---
:mortar_board: [Начинаем программировать](https://up.htmlacademy.ru/react/16/user/2547343/tasks/5)

faviru committed:
Созданы компоненты страниц

baileys-li commented on src/components/App.tsx:
```diff
@@ -0,0 +1,13 @@
+import MainScreen from '../pages/main';
```
У вас есть критерий про названия файлов с маленькой буквы
(мне самому непривычно, но лучше поправить сейчас)

faviru replied:
Точно, спасибо)

baileys-li replied:
Не вышло обновить файл. Такое случается на Windows. Попробуй закоммитить удаление, а потом закоммить новый файл с таким же контентом, но с маленькой буквы

faviru committed:
Исправлено имя файла и добавлены новые компоненты

baileys-li commented on src/components/header/header.tsx:
```diff
@@ -0,0 +1,39 @@
+type HeaderProps = {
+  isNavigation: boolean;
```
Nit: лучше читается кмк
```suggestion
  hasNavigation: boolean;
```

baileys-li commented on src/components/App.tsx:
```diff
@@ -0,0 +1,15 @@
+
+function App(props: AppProps): JSX.Element {
+  return (
+    <MainScreen placeCardsCount={props.placeCardsCount} cities={props.cities} isNavigation={props.isNavigation} />
```
Лучше деструктуизировать
```suggestion
function App({placeCardsCount, cities, isNavigation}: AppProps): JSX.Element {
  return (
    <MainScreen placeCardsCount={placeCardsCount} cities={cities} isNavigation={isNavigation} />
```

baileys-li commented on src/components/locations-list/locations-list.tsx:
```diff
@@ -0,0 +1,18 @@
+}
+
+function LocationsList({ cities }: LocationItemProps) {
+  const List = cities.map((item: string) => <LocationItem city={item} key={item} />);
```
Ну это не класс, и не FC
```suggestion
  const list = cities.map((item: string) => <LocationItem city={item} key={item} />);
```

baileys-li commented on src/components/App.tsx:
```diff
@@ -2,11 +2,13 @@ import MainScreen from '../pages/main';
+  isNavigation: boolean;
 }
 
-function App({ placeCardsCount }: AppProps): JSX.Element {
```
Не вышло обновить файл, скорее всего из-за того, как гит на винде не реагирует на регистр. Попробуй сначала закоммитить удаление этого файла, а потом закоммитить такой же файл на с другим названием

baileys-li reviewed (changes requested):
Нужно ещё связать ПР с заданием в интерфейсе академии

faviru committed:
Удален app.tsx

faviru committed:
Добавлен app.tsx, правки по пуллреквесту

Добавлен app.tsx, исправлены названия переменных, неприкаянные компоненты распределены по директориям
